ci-operator/config/openshift/lightspeed-console/openshift-lightspeed-console-main.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  nodejs-16:
+    name: nodejs-16
+    namespace: openshift
+    tag: latest
+  ubi-9-nginx-120:
+    name: nginx
+    namespace: openshift
+    tag: 1.20-ubi9
+build_root:
+  image_stream_tag:
+    name: nodejs-16
+    namespace: openshift
+    tag: latest
+images:
+- dockerfile_path: Dockerfile
+  from: ubi-9-nginx-120
+  inputs:
+    nodejs-16:
+      as:
+      - build
+  to: ols-console
```

Comment:
Is `ols-console` correct? I didn't see this name used elsewhere.

Reply:
hm, good catch, i think lightspeed-console-plugin is what this should be, i'll fix it.